### DISCUSSION TITLE
Read more option when the text message is too lengthy.

### DISCUSSION THREAD

**Discussion by toshanmugaraj** [element-hq/element-meta#1876] [Ideas]
**Is your feature request related to a problem? Please describe.**
Better user experience when scrolling through old messages. 

**Describe the solution you'd like**
Fix a maximum optimal height for the text message by default, and provide a read more option button at the bottom of each text message when it exceeds the optimal height.

**Comment by toshanmugaraj** [comment]
<img width="207" alt="ima_71433fb" src="https://github.com/vector-im/element-x-ios/assets/665295/f82b5f45-a3bb-4c8c-924e-5bdc1e845ebb">

**Comment by kittykat** [comment]
I'm going to move this to Element-meta discussions because it needs to be a cross-platform product decision
